tests/drivers/clock_control/nrf_lf_clock_start/src/main.c

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2020, Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <ztest.h>
+#include <hal/nrf_clock.h>
+#include <drivers/clock_control/nrf_clock_control.h>
+
+static nrf_clock_lfclk_t type;
+static bool on;
+static uint32_t rtc_cnt;
+
+static void xtal_check(bool on, nrf_clock_lfclk_t type)
+{
+	if (IS_ENABLED(CONFIG_SYSTEM_CLOCK_NO_WAIT)) {
+		zassert_false(on, "Clock should be off");
+	} else if (IS_ENABLED(CONFIG_SYSTEM_CLOCK_WAIT_FOR_AVAILABILITY)) {
+		bool is_running =
+			rtc_cnt | (on && (type == NRF_CLOCK_LFCLK_RC));
```

Comment:
so for essential type this should be `(rtc_cnt != 0) || (on ...`

Reply:
done